packages/client-react-streaming/package.json

```diff
@@ -128,6 +131,7 @@
     "react": "18.3.0-canary-60a927d04-20240113",
     "react-dom": "18.3.0-canary-60a927d04-20240113",
     "rimraf": "5.0.5",
+    "superjson": "1.13.3",
```

Comment:
Version 2 of `superjson` is ESM-only, so for testing this with CJS we have to use version 1